tests/test_nanoevents_physlite.py

```diff
@@ -39,3 +45,91 @@ def test_electron_track_links(events, do_slice):
                     event.GSFTrackParticles[track_index].z0
                     == trackParticles[i][j][link_index].z0
                 )
+
+
+def mock_empty(form, behavior={}):
```

Comment:
@SamKelson You don't want mutable defaults in function signatures (google "python mutable defaults" to see why. These things should be written like

```python
def mock_empty(form, behavior=None):
    behavior = behavior if behavior is not None else {}
```

Reply:
Okay, thanks! I see my mistake haha. [example for myself](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)

Reply:
I will add this to a new PR in addition to making an issue for bumping the uproot version.

Reply:
Yeah, everyone has done this at some point in their life (it is one of those surprising behavior of Python that the first time you hit it you do a lot of head scratching) so no worries. :) 

> I will add this to a new PR in addition to making an issue for bumping the uproot version.

SGTM.